### ISSUE TITLE
Add ability to email PDF

### ISSUE THREAD

**Issue by dodgees** [opennebraska/renters-help#24]

**Comment by chandlerbrtek** [comment]
This definitely seems like an interesting problem. From my short research it looks like something that maybe can only be done on server side? <a/> mailto doesn't allow attachments. Did you have any good ideas of where to start here?

**Comment by dodgees** [comment]
No. I haven't looked into it at all.

**Comment by ronaldewatts** [comment]
From what I understand there is no way to do this without a server-side interaction.

**Comment by chandlerbrtek** [comment]
Yeah that's what it looks like to me as well, only other solution was this: https://www.emailjs.com/, which seems interesting but is not free.
